src/app/components/utility/modal/modal.component.ts

```diff
@@ -4,6 +4,8 @@ import { GlobalService } from '../../../services/global.service';
 import { InputComponent } from '../input/input.component';
```

Comment:
@Suryansh5545 Please undo all the changes of this file as we don't need them for this PR

Reply:
done